Classes/Private/Manager/OCTManager.m

```diff
@@ -74,12 +74,7 @@ - (instancetype)initWithConfiguration:(OCTManagerConfiguration *)configuration
         savedData = [NSData dataWithContentsOfFile:savedDataPath];
     }
 
-    _tox = [[OCTTox alloc] initWithOptions:configuration.options savedData:savedData error:error];
-
-    if (! _tox) {
-        return nil;
-    }
-
+    _tox = [[OCTTox alloc] initWithOptions:configuration.options savedData:savedData error:nil];
```

Comment:
It looks like bad resolve of merge conflict. It should be vice versa, the red code is what we need.